site/src/components/WorkspaceActions/constants.ts

```diff
@@ -0,0 +1,90 @@
+// all the possible states returned by the API
+export enum WorkspaceStateEnum {
+  starting = "Starting",
+  started = "Started",
+  stopping = "Stopping",
+  stopped = "Stopped",
+  canceling = "Canceling",
+  canceled = "Canceled",
+  deleting = "Deleting",
+  deleted = "Deleted",
+  queued = "Queued",
+  error = "Error",
+  loading = "Loading",
+}
+
```

Comment:
I'm surprised we don't have an API type for this. I had to discover all of these states through trial and error.

Reply:
There was a discussion about moving the "computing state" into the backend but I didn't find it. We probably are going to make that at some point.